dashboard/src/pages/TreeDetails/Tabs/TestCards.tsx

```diff
@@ -19,11 +23,37 @@ import { groupStatus } from '@/utils/status';
 import ColoredCircle from '@/components/ColoredCircle/ColoredCircle';
 import { TIssue } from '@/types/general';
 import { NoIssueFound } from '@/components/Issue/IssueSection';
+import { useDiffFilterParams } from '@/utils/filters';
 
 interface IConfigList extends Pick<TTreeTestsData, 'configStatusCounts'> {
   title: IBaseCard['title'];
 }
 
+const LinkWrapper = ({
```

Comment:
Since these link wrappers are completely equal, you can create only one in something like `Pages/TreeDetails/TreeDetailsFilterLink.tsx` with the name `TreeDetailsFilterLink` and import them. 

If there is a slightly difference between them is okay to have more than one and keep them inside the component, but they seem to be completely equal, so you can just reuse a component